core/templates/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -541,6 +543,23 @@ export class ExplorationEngineService {
     questionHtml = questionHtml + this._getRandomSuffix();
     nextInteractionHtml = nextInteractionHtml + this._getRandomSuffix();
 
+    // Check if the answer is only misspelled and close
+    // to the correct answer. Only to be checked if the
+    // interaction is TextInput.
+    let oldInteractionId = oldState.interaction.id;
+
+    // Work => let catchMisspellingsFeatOn =
+    // oldStateCard.getInteraction().customizationArgs.
```

Comment:
Is this supposed to be here?

Reply:
Actually, this feature needs to be enabled when catchMisspellings will be true. The work for the customization_arg is in another large PR that is under review currently.

Reply:
Actually, it has to be done in the PR because further done changes are to be triggered if the catchMisspellings field is true. Haven't made the complete change because it needs the merging of [PR](https://github.com/oppia/oppia/pull/16077) first.